module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -3487,4 +3548,88 @@ protected function checkTitleHoldRules(array $bib, array $patron): bool
         }
         return false;
     }
+
+    /**
+     * Gets title information for holds placed in an INN-Reach system
+     *
+     * @param $holdId the id of the hold from Sierra
+     * @param $itemId the id of the item or bib from Sierra
+     *
+     * @return array
+     *
+     * @throws ILSException
+     */
+    private function getInnReachHoldTitleInfoFromId($holdId, $itemId)
+    {
+        $titleInfo = [];
+        try {
+            $query = 'SELECT 
+                        bib_record_property.best_title as title,
+                        bib_record_property.best_author as author,
+                        --hold.status, -- this shows sierra hold status not inn-reach status
+                        bib_record_property.best_title_norm as sort_title
+                    FROM
+                        sierra_view.hold,
+                        sierra_view.bib_record_item_record_link,
+                        sierra_view.bib_record_property
+                    WHERE
+                        hold.id = $1
+                    AND hold.is_ir=true
+                    AND hold.record_id = bib_record_item_record_link.item_record_id
+                    AND bib_record_item_record_link.bib_record_id = bib_record_property.bib_record_id';
+            pg_prepare($this->db, 'prep_query', $query);
+            $results = pg_execute($this->db, 'prep_query', [$holdId]);
+            while ($resultArray = pg_fetch_row($results)) {
+                $titleInfo = [
+                    'id' => $itemId,
+                    'title' => $resultArray[0],
+                    'author' => $resultArray[1],
+                ];
+            }
+        } catch (\Exception $e) {
+            $this->throwAsIlsException($e);
+        }
+
+        return $titleInfo;
+    }
+
+    /**
+     * Gets title information for checked out items from INN-Reach systems
+     *
+     * @param $checkOutId the id of the checkout from Sierra
+     * @param $itemId     the id of the item or bib from Sierra
+     *
+     * @return array|void
+     *
+     * @throws ILSException
+     */
+    private function getInnReachCheckoutTitleInfoFromId($checkOutId, $itemId)
```

Comment:
Please change private methods to protected -- this makes it easier to locally extend/override the code. (We don't have any private methods in the code base for this reason).